rotkehlchen/chain/ethereum/modules/adex/utils.py

```diff
@@ -145,7 +145,14 @@ def deserialize_adex_event_from_db(
             )
 
         try:
-            token = EthereumToken(event_tuple[13])   # type: ignore # type already checked
+            token = EthereumToken.from_identifier(
+                event_tuple[13],   # type: ignore # type already checked
+            )
+            if token is None:
+                raise DeserializationError(
+                    f'Unknown {event_tuple[13]} found while processing adex event. '
```

Comment:
In the review's suggestion I also had a change in this string:

```suggestion
                    f'Unknown token {event_tuple[13]} found while processing adex event. '
```

Reply:
While doing it I also noticed the entire try/except also goes away. Since it's handling the same error, and the EthereumToken.from_identifier() does not raise, but returns None

Reply:
@LefterisJP Yes, I went for a walk and noticed mid way. I came to fix it but you already did it :sweat_smile: